src/modules/users/actions.js

```diff
@@ -84,3 +84,9 @@ export function create(object) {
   }
 }
 
+export function fetchSuccess(users) {
+  return (dispatch) => {
+    const formatted = users.map(d => _.pick(d, ['auth0_id', 'id', 'name', 'picture']))
```

Comment:
the ``allowedAttributes`` or whatever should be a new function at this point.  It's repeated above.